src/Compiler/Input/AbstractSectorDataReader.cs

```diff
@@ -15,12 +15,23 @@ public bool IsBlankLine(string line)
             return line.Trim() == "";
         }
 
+        /*
+         * Returns whether or not the line is an arc gen line
+         */
+        public bool IsArcGenLine(string line) {

```

Comment:
I'm wondering if there's an alternate way to express arcs that don't require them to be comments 🤔 I think if we start to make some comments "special", things could get confusing.

I think we could look at creating a macro expression (I hate macros, but they're great for this), that gets expanded. That way, we don't have this situation where some comments are special.

Perhaps something like `@ARC(arc def here)`?

For a first pass, we can just parse macros in the parser, but perhaps in the future we can implement something that expands all macros on a line, before we send things over to the parser?

LMK what you think :)

Reply:
Yeah, I was also thinking the syntax of the arcs could be changed, I was just going off of what went into the ArcGen tool. Eventually yeah I could also write a macro expander - I had a look in the rest of the replacers but they were only for options in the file.